legal-api/src/legal_api/resources/v2/business/business.py

```diff
@@ -212,3 +214,18 @@ def search_businesses():
         current_app.logger.info(err)
         current_app.logger.error('Error searching over business information for: %s', identifiers)
         return {'error': 'Unable to retrieve businesses.'}, HTTPStatus.INTERNAL_SERVER_ERROR
+
+
+@bp.route('/allowable/<string:business_type>/<string:business_state>', methods=['GET'])
+@cross_origin(origin='*')
+@jwt.requires_auth
+def get_allowable_for_business_type(business_type: str, business_state: str):
+    """Return a JSON object with information about what a user could theoretically file for a business type."""
+    business_state = business_state.upper()
+    business_type = business_type.upper()
+    if business_type.startswith('T'):
+        return {'message': babel('No information on temp registrations.')}, 200
```

Comment:
it is identifier which starts with T. you can remove this validation

Reply:
Removed

Reply:
@vysakh-menon-aot Why should this validation be removed?

Reply:
this endpoint doesn't have identifier as parameter

Reply:
Oh, right 😁 Thanks.